vis-clipboard

```diff
@@ -76,27 +80,27 @@ vc_paste() {
 }
 
 vc_wlclipboard_copy() {
-	wl-copy -t TEXT
+	wl-copy --"$sel" -t TEXT
 }
 
 vc_wlclipboard_paste() {
-	wl-paste -t text
+	wl-paste --"$sel" -t text
```

Comment:
Does your `wl-{copy,paste}` actually support a `--clipboard` option? It seems to be the default and only `--primary` is recognized.

Reply:
Thanks! That does not appear to be the case.
I appreciate your feedback.